### DISCUSSION TITLE
Form 的属性previewTextPlaceholder如果没有设置的话 应该继承全局上下文的值吧

### DISCUSSION THREAD

**Discussion by Nokecy** [alibaba/formily#1721] [Q&A 2.x] [answered]
https://codesandbox.io/s/elegant-sun-46s9x?file=/App.tsx

**Comment by janryWang** [comment, marked as answer]
Form组件内部本身套了个Placeholder，被拦截了，你把组件放在Form内部就行了

**Reply by Nokecy** [reply]
没有设置Form的previewTextPlaceholder的时候，是不是应该继承全局上下文的值呢~~
